src/profiles-pycorelib/attribution_model.py

```diff
@@ -0,0 +1,239 @@
+# Commenting out this model as this has some breaking changes for v0.10.4
+# This will be fixed in the next release
+from typing import List, Dict, Tuple, Union
+
+from profiles_rudderstack.model import BaseModelType
+from profiles_rudderstack.schema import ContractBuildSpecSchema, EntityKeyBuildSpecSchema, EntityIdsBuildSpecSchema, MaterializationBuildSpecSchema
+from profiles_rudderstack.recipe import PyNativeRecipe
+from profiles_rudderstack.material import WhtMaterial
+from profiles_rudderstack.logger import Logger
+import pandas as pd
+import numpy as np
+import ast
+
+class AttributionModel(BaseModelType):
+    TypeName = "campaign_attribution_scores" # the name of the model type
+
+    # json schema for the build spec
+    BuildSpecSchema = {
+        "type": "object",
+        "properties": {
+            "touchpoint_var" : { "type": "string" }, # model
+            "days_since_first_seen_var": { "type": "string" }, # model
+            "first_seen_since": { "type": "integer" },
+            "conversion_entity_var": { "type": "string" }, #model
+            "output_field": { "type": "string" },          
+            "entity": { "type": "string"}
```

Comment:
```suggestion
            **EntityKeyBuildSpecSchema["properties"],
```
Suggested to use the schemas provided for standard wht properties

Reply:
Can ignore if this isn't used for declaring the entity key, might've misunderstood it

Reply:
Yes, it's not to declare the entity, but knowing what the entity is and use that to refer to the entity var model